index.html

```diff
@@ -235,14 +235,12 @@ <h2>Software projects</h2>
             as the map rendering engine.
           </p>
         </li>
-        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML Community Module</a>
+        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML Extension</a>
```

Comment:
Same question as https://github.com/Maps4HTML/MapML/pull/170#discussion_r620016978, should this rather be <q>GeoServer MapML Extension Module</q>?